src/components/editor/port.tsx

```diff
@@ -29,7 +29,7 @@ const EditorPort: FunctionComponent<PortProps> = memo(function WrappedPort({
 			id={ port.id }
 			position={ handlePositionByPortDirection[port.direction] }
 			data-c74-type={ port.type }
-			data-c74-name={ alias || port.id }
+			data-c74-name={ (alias || port.id).replace(/\((capture|playback)_[0-9+]\)/, "") }
```

Comment:
Hmm.. so, if a port has no alias and is just called `"capture_1"` then it just becomes `""` ?

Reply:
ah, I think I tried only applying the regex to alias and was getting an error, I'll look at that more closely.

Reply:
Oh whoops I had it flipped around. Does this work? 
`data-c74-name={ (alias || port.id.replace(/\((capture|playback)_[0-9+]\)/, "")) }`

Reply:
I suspect we'll wanna remove the `[0-9]+` from the regex but i'll have to test it out

Reply:
Just so you know, the intention was to only remove the labels added by pipewire-jack containing capture or playback followed by an underscore and a number, enclosed in parentheses. Just trying to prettify the names and remove redundant information. Hopefully I got the regex right and hopefully this is not a pattern that would be commonly used for any other reason.